Fight_Game

S-ishita opened the pull request:
## PR Description 📜

Please include summary related to the issue you have fixed and describe your PR in brief over here by specifying the issue number on which you were working below
Fixes #52 

<hr>
 
## Mark the task you have completed ✅

<!----Please delete options that are not relevant. In order to tick the check box just but x inside them for example [x] like this----->

- [x] I follow [CONTRIBUTING GUIDELINE](https://github.com/kunjgit/GameZone/blob/main/.github/CONTRIBUTING_GUIDELINE.md) & [CODE OF CONDUCT](https://github.com/kunjgit/GameZone/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [x] I have performed a self-review of my own code or work.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generates no new warnings.
- [x] I have followed proper naming convention showed in [CONTRIBUTING GUIDELINE](https://github.com/kunjgit/GameZone/blob/main/.github/CONTRIBUTING_GUIDELINE.md)
- [x] I have added screenshot for website preview in assets/images 
- [x] I have added entries for my game in main README.md
- [x] I have added README.md in my folder 
- [x] I have added working video of the game in README.md (optional)
- [x] I have specified the respective issue number for which I have requested the new game.


<hr>

## Add your screenshots(Optional) 📸


![f](https://github.com/kunjgit/GameZone/assets/104543751/502d3587-4a3a-4a00-a82b-996d09ecb792)
![f1](https://github.com/kunjgit/GameZone/assets/104543751/4e0362ef-bb71-44fa-811c-58152491dcc2)


--- 
<br>

## Thank you soo much for contributing to our repository 💗

rajatnai49 commented:
Hey, @S-ishita please complete all the task which checklist you have given please read all the task and try to complete all of them, and when you complete all the given task then you can see your game on home page of the website like,

![Screenshot from 2023-05-21 09-55-18](https://github.com/kunjgit/GameZone/assets/96529504/4219a7da-a72e-48b8-bab1-3812234ce568)

You can see given games, so you get idea that which are files you have to update!!

S-ishita commented:
Hey @rajatnai49 . I updated the file now.Sorry for the mistake from my end.

S-ishita commented:
- [x] I have added screenshot for website preview in assets/images
- [x] I have added entries for my game in main README.md
- [x] I have added README.md in my folder

kunjgit commented on Games/Fight_Game/README.md:
```diff
@@ -0,0 +1,4 @@
+# Fight-game
+
+#### Player 1 Controls:W A D Space F
+#### Player 2 Controls:ArrowUp ArrowLeft ArrowRight ArrowDown Control
```
add proper readme

kunjgit commented on README.md:
just remove the number and add * in place of the number !!

S-ishita replied:
done

kunjgit commented:
don't alter main index file and also make sure if you are adding game in main Readme you have to have your game under all existing games and you can have game number  as * we will add that

S-ishita commented:
@kunjgit following changes have been made:

- undo the index.html file changes
- changes the number to * in readme

kunjgit reviewed (changes requested):
make sure you don't alter main index file we have changed few things make it up to date to latest and don't alter it

kunjgit commented:
Thank you  @S-ishita , for your valuable time and contribution in our GameZone 💗. 
 It’s our GameZone, so Let’s build this GameZone altogether !!🤝
  Hoping to see you soon with another PR again 😇 
 Wishing you all the best for your journey into Open Source🚀
